tests/kiwi_images_test/kiwi_boot.pm

```diff
@@ -21,37 +21,35 @@ use strict;
 use warnings;
 use testapi;
 use utils;
-use version_utils "is_sle";
 
 sub run {
-    my $install_type = get_var('KIWI_OLD');
-    my $iso_name     = get_var('ISO');
+    my $iso_name = get_var('ISO');
     # bootloader screen is too fast to openqa
     sleep(10);
-    send_key 'down';
-    send_key 'up';
-    # perl kiwi oem installer does not work correctly on non interactive
-    if (($install_type == 1) && ($iso_name =~ /OEM/)) {
+    if ($iso_name =~ /OEM/) {
+        send_key 'up';
+        send_key 'down';
         assert_screen('kiwi_boot', 15);
         send_key 'ret';
-        # choose last option on installer screen
-        assert_screen('kiwi_oem_install_installer', 300);
-        my $count = 0;
-        while ($count < 16) {
-            send_key 'down';
-            $count++;
+        # SLE12SP3 image contains a interactive installer
+        if ($iso_name =~ '12-SP3') {
```

Comment:
Can you please use /12-SP3/ instead '12-SP3' for the match, as on line 29 ?
It's expected for regex.